materials/data-structures-R.md

```diff
@@ -72,12 +82,37 @@ surveys <- data.frame(sites, counts, weights)
     * `surveys[c("counts", "weights")]`
     * `surveys$sites`
     * `surveys[["sites"]]`
-* Subsetting rows: 
-    * `subset(surveys, sites == 'a')`
-* Subsetting values/blocks: 
-    * `surveys[1,2]` 
-    * `surveys[1:2,1:3]`
 
-> Do [Exercise 3 - More Variables]({{ site.baseurl }}/exercises/Expressions-and-variables-more-variables-R/).
+
+### Importing data
+
+* `read.csv()`
+* Download the file for the Shrub Volume 2 exercise
+* How do we tell R where this file is
+
+> When someone says `setwd` discuss issues
```

Comment:
Could you include the `setwd()` issues you hope to discuss? I tend to use the `'Session' -> 'Set Working Directory'` menu option. Perhaps, I should reconsider.